apps/vault/test/vault.js

```diff
@@ -230,6 +230,39 @@ contract('Vault app', (accounts) => {
         assert.equal((await token.balanceOf(vault.address)), 10, 'vault should have 10 balance')
         await assertRevert(() => vault.transferToVault(ETH))
       })
+
+      it('does not allow recoverability', async () => {
+        assert.isFalse(await vault.allowRecoverability(token.address))
+        assert.isFalse(await vault.allowRecoverability(ETH))
```

Comment:
Would also be great to test the entire flow, trying to actually recover funds from the vault.

Reply:
This is done above:
https://github.com/aragon/aragon-apps/pull/435/files#diff-4182395b6bdaf5bbad54a4607aa62a87R208
https://github.com/aragon/aragon-apps/pull/435/files#diff-4182395b6bdaf5bbad54a4607aa62a87R217
I  just wanted to add the explicit call to that function just in case and for comparison with the non-init case.